.github/workflows/multiarch.yml

```
@@ -0,0 +1,19 @@
+---
+name: Multi-arch builds
```

```suggestion
name: Multi-arch Builds
```

Hate to be so nitty, but the consistent UI is nice.